experiments/tester/src/RNTester/TestComponents/Common/Slider.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';

```

Comment:
Ignore this file. I have sent out a separate PR for this change. Including it here to pass the CI build.